requirements.txt

```diff
@@ -4,3 +4,5 @@ six==1.9.0
 django-bootstrap3==5.4.0
 djangorestframework==3.1.3
 djangorestframework-jwt==1.6.0
+psycopg2==2.6.1
+configparser >= 3.5.0b1
```

Comment:
Whats configparser used for in this context?  If its the ConfigParser module we don't need it because it should be part of a standard python installation.